resources/select2/_multiple.scss

```diff
@@ -81,6 +81,10 @@
 }
 
 &.select2-container--focus {
+    width: 100%;
```

Comment:
Let's ... not change values here, as they affect _all_ select2 widgets across the entire site. If they are necessary, add them in `problem.scss` inside i.e. `#problem-submit`.